doc/source/visualizing/geographic_projections_and_transforms.rst

```diff
@@ -0,0 +1,145 @@
+.. _geographic_projections_and_transforms:
+
+Geographic Projections and Transforms
```

Comment:
We will eventually -- not as part of this PR -- make note that this is also the case for what folks in the astro simulation community might just think of as spherical data.